incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/openapi/model/Operation.java

```diff
@@ -22,4 +22,5 @@ public class Operation
     public List<Map<String, List<String>>> security;
     public RequestBody requestBody;
     public List<Parameter> parameters;
+    public Map<String, ResponseByContentType> responses; // responses by status code
```

Comment:
Please remove the comment.
If you feel it is not clear enough, rename `responses` to `responsesByStatus`.

Reply:
removed